nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/action/computation/AbstractTransientBlobComputation.java

```diff
@@ -61,8 +62,8 @@ protected String getTransientStoreKey(String commandId) {
         return metadata.name() + commandId;
     }
 
-    public Blob getBlob(String key) {
-        TransientStore store = Framework.getService(TransientStoreService.class).getStore(CSVExportAction.ACTION_NAME);
+    public Blob getBlob(String key, String storeName) {
```

Comment:
There doesn't seem to be any javadoc in this file.  I realise you are just changing a method signature.